docs/changelog.md

```diff
@@ -15,11 +15,13 @@ Bug fixes
 - Fixed an issue where `pyscript` would not be available when using the minified version of PyScript. ([#1054](https://github.com/pyscript/pyscript/pull/1054))
 - Fixed missing closing tag when rendering an image with `display`. ([#1058](https://github.com/pyscript/pyscript/pull/1058))
 - Fixed a bug where Python plugins methods were being executed twice. ([#1064](https://github.com/pyscript/pyscript/pull/1064))
+- Fixes [#1280](https://github.com/pyscript/pyscript/issues/1280).
```

Comment:
Do we want to explain it more?

I think these additions shouldn't be added to the previous release 😄  @WebReflection  created a new entry for the future release